listenbrainz/webserver/views/user.py

```diff
@@ -424,7 +424,7 @@ def delete_user(musicbrainz_id):
     """
 
     user = _get_user(musicbrainz_id)
-    timescale_connection._ts.delete(user.musicbrainz_id)
+    timescale_connection._ts.delete(musicbrainz_id, user.id)
```

Comment:
we should still use `user. musicbrainz_id` here, in case there are upper/lower-case differences between the username and what's in redis

Reply:
Something weird is going on here, this function receives the musicbrainz_id not from what the user submitted but from a db query and then we do a db query here again. Will look deeper later for now changing user.musicbrainz_id

Reply:
yeah, I wouldn't be surprised if we go username -> look up user -> call this method -> lookup again. 
Making a ticket to pass in a User object here sounds good

Reply:
LB-1055